database_wrapper.py

```diff
@@ -565,13 +577,22 @@ def insert_entity(self, entity_type, data_dict: dict) -> bool:
         entity = entity_type()
 
         # Logging...
-        print("{}Inserting into {}...{}".format(
-              CYAN_COLOR_CODE, entity_attributes["__tablename__"], RESET_COLOR_CODE))
+        print(
```

Comment:
I prefer the old formatting style but I'll just grumble about it for now since black likes it this way.

Reply:
Agreed, but I've gotta choose my battles wisely 😆